sysl2/sysl/seqs.go

```diff
@@ -0,0 +1,1081 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/anz-bank/sysl/src/proto"
+	"github.com/sirupsen/logrus"
+)
+
+type indentWriter struct {
+	bytes.Buffer
```

Comment:
Generally speaking, prefer the more generalised type (`io.Writer` in this instance).

This would mean explicitly holding a pair of `bytes.Buffer` in `sequenceDiagWriter` and therefore may not be warranted for this narrowly scoped usage.